sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -157,13 +151,25 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     return await this.save(institution);
   }
 
+  async syncInstitution(userInfo: UserInfo): Promise<void> {
+    const account = await this.bceidService.getAccountDetails(
+      userInfo.idp_user_name,
+    );
+    const institutionEntity = await this.getInstitute(userInfo);
+    institutionEntity.guid = account.institution.guid;
```

Comment:
Please consider not updating the institution guid and, please add validation to check if the institution guid on account object is the same institution guid retrieved from DB to be updated.

Reply:
DONE